dataportal/muxer/data_muxer.py

```
@@ -258,6 +261,12 @@ def _process_new_descriptor(self, descriptor):
                                          name, self.sources[name],
                                          descriptor.id,
                                          description['source']))
+                if name == 'time':
+                    # We can argue later about how best to handle this corner
+                    # case, but anything is better than silently mislabeling
+                    # data.
+                    raise ValueError("The name 'time' is reserved and cannot "
```

:+1: for failing early and hard.